### ISSUE TITLE
Not able to create .bsprof file 

### ISSUE THREAD

**Issue by puneetamb** [rokuality/rokuality-js#10]
Hi Team,

Getting error while capturing profiling data.
Error: 

   ```
 ENOENT: no such file or directory, open '/var/folders/13/t9q7wfjj0gz907xtrk5vgvt80000gn/T/com.rokuality/9zuz38zvl89fjknrq80omfcf4ije5b9njs165aw4.bsprof'

      110 |
      111 |       var filePath = rokualityTempDir + path.sep + (await this.generateRandomString(40)) + fileExtension;
    > 112 |       fs.writeFileSync(filePath, fileData);
      113 |       return filePath;
      114 |     }
      115 |   }, {
      
      at JSONUtils.convertToFile (../build/utils/FileToStringUtils.js:112:10)
      at async RokuInfo.getPerformanceProfile (../build/driver/roku/RokuInfo.js:125:14)
      at async Object.<anonymous> (roku.test.js:109:23)
```

**Comment by rokualitysupport** [comment]
Most likely a duplicate of https://github.com/rokuality/rokuality-js/issues/11

**Comment by rokualitysupport** [comment]
Please try new Rokuality version 1.3.2 available at https://www.rokuality.com/download which on start will help manage those local user temp directories and should prevent this issue.

**Comment by rokualitysupport** [comment]
This is now fixed client side in latest js version 1.5.2

**Comment by puneetamb** [comment]
Hi Team,
Took the latest code(1.5.2) and downloaded latest rokuality app (1.3.2). But still profile data capturing is failing.

`  Performance profiling failed to initiate for device 192.168.1.5!

      17 |     _classCallCheck(this, SessionNotStartedException);
      18 |
    > 19 |     var _this = _possibleConstructorReturn(this, (SessionNotStartedException.__proto__ || Object.getPrototypeOf(SessionNotStartedException)).call(this, message));
      20 |
      21 |     _this.name = _this.constructor.name;
      22 |     return _this;
      
      at new SessionNotStartedException (../build/exceptions/SessionNotStartedException.js:19:142)
      at RokuDriver.handler (../build/driver/roku/RokuDriver.js:183:15)
      at ../build/driver/roku/RokuDriver.js:74:35
      at Object.<anonymous> (roku.test.js:214:22)`

Also, the app remains in running state but test session gets killed on encountering the above error.
